tests/driver_setup/__init__.py

```diff
@@ -48,16 +57,16 @@ def current(cls):
     def clean_args():
         # type: () -> List[str]
         args_to_remove = [
-            '--port-a',
-            '--port-b',
-            '--baud-rate',
-            '--response-timeout',
-            '--retransmission-interval',
-            '--mtu',
-            '--iterations',
-            '--log-level',
-            '--driver-log-level',
-            '--nrf-family'
+            "--port-a",
+            "--port-b",
+            "--baud-rate",
+            "--response-timeout",
+            "--retransmission-interval",
+            "--mtu",
+            "--iterations",
+            "--log-level",
+            "--driver-log-level",
+            "--nrf-family",
```

Comment:
If I remember right, python 2.7 fails with comma at the end.

Reply:
All tests are ran with python 2.7...